tests/foreman/cli/test_contentviews.py

```diff
@@ -315,6 +318,180 @@ def test_delete_cv_version(self):
         result = ContentView.version_delete({u'id': version1_id})
         self.assertEqual(result.return_code, 0)
 
+    def test_remove_cv_environment(self):
+        """@Test: Remove content view from lifecycle environment assignment
+
+        @Feature: Content Views
+
+        @Assert: Content view removed from environment successfully
+
+        """
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': new_cv['id']})
+        result = ContentView.info({u'id': new_cv['id']})
+        env = result.stdout['lifecycle-environments'][0]
+
+        result = ContentView.remove({
+            u'id': new_cv['id'],
+            u'environment-ids': env['id'],
+            u'organization-id': new_org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentView.info({u'id': new_cv['id']})
+        self.assertFalse(result.stdout['lifecycle-environments'])
```

Comment:
:+1: to @elyezer's suggestion. When I see this:

    self.assertFalse(…)

I assume that a boolean is being checked. And when I see this:

    self.assertEqual(len(…), 0)

I assume an integer, sequence or similar is being checked. Using the more specific assertion makes the test easier to understand.

Also, note that PEP 008 was giving that suggestion in the context of flow control, not unit testing. That situations is similar, but not exactly the same as here. Good point, though. I appreciate it.